pkg/kubectl/cmd/edit.go

```diff
@@ -131,7 +131,7 @@ func runEdit(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args
 		return err
 	}
 
```

Comment:
Thanks for the refactor.  The runEdit function is 200 lines long with minimal documentation, which makes it rather hard to understand.  Would you mind throwing a TODO: at the top of the function to refactor this into ~30-50 line pieces.